presets/presets.lua

```diff
@@ -1077,74 +1077,189 @@ presets = {
+            {type="item",       name="automation-science-pack",      quantity=1000, next="x10"},
+            {type="item",       name="logistic-science-pack",        quantity=1000, next="x10"},
+            {type="item",       name="military-science-pack",        quantity=1000, next="x10"},
+            {type="item",       name="chemical-science-pack",        quantity=1000, next="x10"},
+            {type="item",       name="production-science-pack",      quantity=1000, next="x10"},
+            {type="item",       name="utility-science-pack",         quantity=1000, next="x10"},
+            {type="item",       name="space-science-pack",           quantity=10000, next="x10"},
+            {type="item",       name="matter-tech-card",             quantity=1000, next="x10"},
+            {type="item",       name="advanced-tech-card",           quantity=1000, next="x10"},
+            {type="item",       name="singularity-tech-card",        quantity=1000, next="x10"},
+
+            {type="group",      name="Stages"},
+            {type="technology", name="el_stage_tech", quantity=1},
+            {type="technology", name="el_energy_tech", quantity=1},
+            {type="technology", name="fi_stage_tech", quantity=1},
+            {type="technology", name="fi_energy_tech", quantity=1},
+            {type="technology", name="fu_stage_tech", quantity=1},
+            {type="technology", name="fu_energy_tech", quantity=1},
+            {type="technology", name="gr_stage_tech", quantity=1},
+            
+            {type="group",      name="K2 Progress"},
+            {type="item",       name="ai-core",                      quantity=1},
+            {type="fluid",      name="matter",                       quantity=1},
+            {type="technology", name="kr-advanced-pickaxe",          quantity=1},
+            {type="item",       name="kr-intergalactic-transceiver", quantity=1},
+
+            {type="group",      name="K2 Resources"},
+            {type="fluid",      name="mineral-water",                quantity=1},
+            {type="item",       name="silicon",                      quantity=1},
+            {type="item",       name="rare-metals",                  quantity=1},
+            {type="item",       name="imersite-powder",              quantity=1},
+
+            {type="group",      name="vanilla"},
+            {type="fluid",      name="petroleum-gas",                quantity=1},
+            {type="item",       name="locomotive", quantity=1},
+            {type="item",       name="construction-robot", quantity=1},
+            {type="item",       name="logistic-robot", quantity=1},
+            
+            {type="group",      name="Electronics"},
+            {type="item",       name="el_materials_ALK", quantity=1},
+            {type="item",       name="el_energy_crystal_item", quantity=1},
+            {type="item",       name="el_diesel_train_item", quantity=1},
+
+            {type="group",      name="Fisson"},
```

Comment:
typo

Reply:
It's "Fission" right?

Reply:
yeah thanks for cathing that im blind as it turns out

Reply:
fixed